packages/client/src/client4.ts

```diff
@@ -481,6 +476,10 @@ export default class Client4 {
         return `${this.getBaseRoute()}/drafts`;
     }
 
+    getBoardsRoute() {
+        return this.useBoardsProduct ? '/plugins/boards/api/v2' : '/plugins/focalboard/api/v2';
```

Comment:
nit: Could use `getPluginsRoute`

Reply:
Good point. I think I might leave it as-is though because there'd been some discussion about moving those APIs since having them under `/plugins` is a bit weird.